### DISCUSSION TITLE
Vocês conhecem o gitkraken?

### DISCUSSION THREAD

**Discussion by dsoldera** [frontendbr/forum#265] [Perguntas e Respostas]
Olá pessoal.

Vcs conhecem o gitkraken?? O que acham dele?

http://www.gitkraken.com

[]s,

**Comment by akfzambrana** [comment]
Não acha melhor utilizar a linha de comando? Eu costumo utilizar o git via terminal, mesmo no Windows :wink:

**Comment by herberthenrique** [comment]
Eu vi, achei bem legal.
Mas vou ser sempre fã do terminal, não tem jeito hahaha

**Comment by renatoagds** [comment]
@dsoldera usei ele algumas vezes e achei super bacana, o único ponto é que eu achei ele um pouco pesado, por isso prefiro usar no CLI direto. 

Ele tem suporte a diversas coisas, inclusive ao Git Flow, além disso eles estão atualizando constantemente, a cada uma ou duas semanas rola um deploy deles. 

Acho uma boa alternativa para Linux, para Mac e Win ainda acho válido o kit da Atlassian.

**Comment by felipefialho** [comment]
Eu dei uma olhada faz um tempinho, é uma alternativa para quem não tem intimidade com linha de comando (especialmente Designers e afins).

Mas acho muito mais rápido e prático o bom e velho terminal.

**Comment by rubenmarcus** [comment]
TortoiseGit para merges e rebases e terminal pra push/pull .. é o que eu uso aqui

**Comment by notnotgabriel** [comment]
Eu gosto do GitKraken, ele é tão completo que nem parece ser free.
É bom pra ver o source tree e fazer diff, mas tenho medo de clicar nas coisas e sair mergeando e commitando (Sim ele faz TUDO mesmo).
Para coisas do dia a dia o terminal traz mais segurança eu acho.

**Comment by felipefialho** [comment]
hahaha, pode cre @gabriel-ribeiro-ir

**Comment by renatoagds** [comment]
@gabriel-ribeiro-ir interessante que eu sempre senti o medo inverso, por vir do viés de Design, sempre preferi usar algo com UI, então sempre tive medo de usar no CLI, até que mudei definitivamente haha

**Comment by wwwflorencio** [comment]
Acho legal, uso as vezes quando quando quero visualizar algumas alterações, comparar e ect

**Comment by adeonir** [comment]
@renatoagds eu também vim do design e sempre tive esse receio ao usar o terminal, mas a cada dia tenho usado mais e mais.

@dsoldera testei o GitKraken e achei até bem bom, mas eu recomendo aguardar o [Git-Tower](https://www.git-tower.com/p/windows-beta), pra mim é o melhor GUI pra Mac.

**Comment by yanmagale** [comment]
Conheço e utilizo, realmente é uma grande tool para UI de projetos GIT.

Claro, nada mais prático do que o terminal. Gosto de usar o Kraken quando estou iniciando uma nova feature com o Git Flow, para poder fazer alguns merges mais complicados, e também para dar aquele refresco a vista, já que a UI dele é bem bacana.

**Comment by jdrgoms** [comment]
É bem por aí mesmo @YanMagale

**Comment by mtmr0x** [comment]
Como eu sempre falo: apenas mais um Chrome consumindo sua memória.

> **GitKraken** is the only Git client built on **Electron**...

e então:

> The **Electron** framework lets you write cross-platform desktop applications using JavaScript, HTML and CSS. It is based on Node.js and **Chromium**...

que no final:

> **Chromium** is an open-source **browser** project...

Conclusão: tenha uma máquina de 16GB de RAM ou use o terminal.
Bom, é isso. :)

**Comment by kazzkiq** [comment]
@matmarsiglio Apps baseados no Chromium consomem muita memória?! Que nada! Isso é coisa da sua cabeça... 😄 

<img width="575" alt="captura de tela 2016-09-13 as 15 14 43" src="https://cloud.githubusercontent.com/assets/1953194/18485380/f435639e-79c2-11e6-91e7-c66bb4502467.png">

_(Obs.: em defesa do ~~indefensável~~ VSCode, eu to com 2 projetos abertos ao mesmo tempo sem fechar o app nem reiniciar a máquina há uns 3 dias)_
